packages/strapi-database/lib/constants/index.js

```diff
@@ -3,7 +3,37 @@
 // contentTypes and components reserved names
 const RESERVED_MODEL_NAMES = ['admin', 'boolean', 'date', 'date-time', 'time', 'upload'];
 // attribute reserved names
-const RESERVED_ATTRIBUTE_NAMES = ['_id', 'id', 'length', 'attributes', 'relations', 'changed'];
+const RESERVED_ATTRIBUTE_NAMES = [
+  '_id',
+  'id',
+  'length',
+  'attributes',
+  'relations',
+  'changed',
+  'created_by',
```

Comment:
We already added created_by and updated_by in features/rbac ;)

Reply:
I chose to let it because it's going to conflict anyway :)

Reply:
sure why not